SWLOR.Game.Server/Feature/AbilityDefinition/Force/DisturbanceAbilityDefinition.cs

```diff
@@ -25,15 +26,51 @@ public class DisturbanceAbilityDefinition : IAbilityListDefinition
             return _builder.Build();
         }
 
-        private void Impact(uint activator, uint target, int dmg, int accDecrease, int tier, string effectTag, int dc)
+        private static void ImpactAction(uint activator, uint target, int level, Location targetLocation)
         {
+            var willBonus = GetAbilityScore(activator, AbilityType.Willpower);
+            var dmg = 0;
+            string effectTag = "";
+            int dc = 0;
+            int accDecrease = 0;
+
+            switch (level)
+            {
+                case 1:
+                    dmg = willBonus;
+                    effectTag = "Tier1Tag";
+                    dc = 8;
```

Comment:
Specifically on the restructuring:

- Add the DMG value to the method signature
- Use the constant name, not a string literal for "Tier1Tag". 
- Add the DC value to the method signature.
- Add the accuracy decrease value to the method signature.